vehicle/mkvi/software/air_control/test_air_control.py

```diff
@@ -0,0 +1,11 @@
+# Base/extended python imports
```

Comment:
@richardli03 Do we still need this file?

Reply:
This will get updated as part of #231. You can remove from this PR.